openscap_daemon/oscap_helpers.py

```diff
@@ -455,7 +474,7 @@ def _get_result_id(results_path):
     return test_result.attrib["id"]
 
 
-def generate_fix_for_result(config, results_path, fix_type):
+def generate_fix_for_result(config, results_path, fix_type, xccdf_id):
```

Comment:
default xccdf_id to None

Reply:
@mpreisler Good catch, I created PR https://github.com/OpenSCAP/openscap-daemon/pull/120